modules/hal_nordic/nrfx/Kconfig

```diff
@@ -573,6 +573,26 @@ config NRFX_UARTE3
 	depends on $(dt_nodelabel_has_compat,uart3,$(DT_COMPAT_NORDIC_NRF_UARTE))
 	select NRFX_UARTE
 
+config NRFX_UARTE_CONFIG_SKIP_GPIO_CONFIG
+	bool "UARTE GPIO configuration support"
+	default y
+
+config NRFX_UARTE_CONFIG_SKIP_PSEL_CONFIG
+	bool "UARTE PSEL configuration support"
+	default y
+
+config NRFX_UARTE_CONFIG_TX_LINK
+	bool "UARTE TX transfer linking support"
```

Comment:
Configs have skip in the name but seem like it's skipping if the Kconfig is `n` not `y`? So the name doesn't really match up

Reply:
Introduced in https://github.com/zephyrproject-rtos/zephyr/pull/65155

Reply:
Are you refering to NRFX_UARTE_CONFIG_SKIP_PSEL_CONFIG? Because comment is on line TX_LINK.
If option is set to `y` then PSEL registers are not touched (are skipped) by the nrfx_uarte driver. We want to have that in Zephyr because of pinctrl.

Reply:
Not answered

Reply:
Diff dropped from this PR.